tests/coverage.sh

```diff
@@ -24,19 +27,30 @@ ADD examples /var/lib/githooks/examples
 RUN git config --global user.email "githook@test.com" && \
     git config --global user.name "Githook Tests"
 
-ADD tests/exec-steps.sh tests/${STEPS_TO_RUN} tests/replace-inline-content.py /var/lib/tests/
+ADD tests/exec-steps.sh tests/${STEPS_TO_RUN} /var/lib/tests/
 
 # Some fixup below:
 # We overwrite the download to use the current install.sh in all scripts
 RUN \\
 # Make sure we're using Bash for kcov
-    find /var/lib -name '*.sh' -exec sed -i 's|#!/bin/sh|#!/bin/bash|g' {} \\; && \\
-    find /var/lib -name '*.sh' -exec sed -i 's|sh /|bash /|g' {} \\; && \\
-    find /var/lib -name '*.sh' -exec sed -i 's|sh "|bash "|g' {} \\; && \\
+    find /var/lib -name '*.sh' -exec sed -i -E 's|#!/bin/sh|#!/bin/bash|g' {} \\; && \\
+    # at the beginnig of line
+    find /var/lib -name '*.sh' -exec sed -i -E 's|^( *)sh /|\1bash /|g' {} \\; && \\
+    find /var/lib -name '*.sh' -exec sed -i -E 's|^( *)sh "|\1bash "|g' {} \\; && \\
+    find /var/lib -name '*.sh' -exec sed -i -E 's|^( *)sh ~/|\1bash /home/coverage/|g' {} \\; && \\
+    # in between line
+    find /var/lib -name '*.sh' -exec sed -i -E 's|( +)sh /|\1bash /|g' {} \\; && \\
+    find /var/lib -name '*.sh' -exec sed -i -E 's|( +)sh "|\1bash "|g' {} \\; && \\
+    find /var/lib -name '*.sh' -exec sed -i -E 's|( +)sh ~/|\1bash /home/coverage/|g' {} \\; && \\
+    # git hooks alias
+    find /var/lib -name '*.sh' -exec sed -i 's|"!sh |"!bash |g' {} \\; && \\
 # Revert changed shell script filenames
     find /var/lib -name '*.sh' -exec sed -E -i "s|/var/lib/githooks/([a-z-]+)\\.bash|/var/lib/githooks/\\1.sh|g" {} \\; && \\
-# Replace the inline content with loading the source file
```

Comment:
I'd need to have a closer look, but doesn't this mean that we now count the inline content twice in the coverage?

Reply:
Oh, we don't have inline content anymore, do we?

Reply:
There is no inline content anymore ;) :-P

Reply:
This is gone a long time ago ;-) fortunately

Reply:
FYI: We made huge progress I think with this code -> we moved towards git based install, we dropped all inline content, we fixed converage. This project underwent some pretty crazy changes, however thanks to the testing it all works! ;)